misc/kafka-util/summarize_topics.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+# Copyright Materialize, Inc. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+"""summarize_topics
+
+Basic script to print messages count and number of bytes within a Kafka topic
+"""
+
+import argparse
+import fnmatch
+import time
+
+import kafka
+
+
+def summarize_topic(args, topic):
+    """Read messages from a topic and print basic information about the topic."""
+
+    consumer = kafka.KafkaConsumer(
+        topic,
+        auto_offset_reset="earliest",
+        consumer_timeout_ms=1000,
+        bootstrap_servers=[f"{args.kafkahost}:{args.port}"],
+        enable_auto_commit=True,
+    )
+
+    start = time.time()
```

Comment:
this should probably be [time.monotonic](https://docs.python.org/3/library/time.html#time.monotonic).

Reply:
They finally did it!

I'll update my code :)